staging/src/k8s.io/apimachinery/pkg/util/managedfields/internal/managedfieldsupdater_test.go

```diff
@@ -335,7 +335,7 @@ func TestTakingOverManagedFieldsDuringUpdateDoesNotModifyPreviousManagerTime(t *
 		},
 		"data": {
 			"key_a": "value",
-			"key_b": value"
+			"key_b": "value"
```

Comment:
huh... these test typos are distressing... did you just notice these and fix them while you were here, or did fixing these reveal that these tests were broken?

Reply:
non-blocking for this PR, but cc @jpbetz for visibility to how accepting yaml parsing is of malformed json input sent as yaml. This was parsing as:

```
{
  "apiVersion": "v1",
  "data": {
    "key_a": "value",
    "key_b": "value\""
  },
  "kind": "ConfigMap",
  "metadata": {
    "name": "configmap"
  }
}
```

Reply:
The former, I just noticed these so fixed them. I was wondering why these typos didn't break the testing, and now I get the answer from your explanations above. Thanks!